src/containers/Tenant/ObjectSummary/ObjectSummary.tsx

```diff
@@ -7,26 +7,20 @@ import qs from 'qs';
 import _ from 'lodash';
 
 import {Button, HelpTooltip, Loader, Tabs} from '@yandex-cloud/uikit';
-//@ts-ignore
```

Comment:
This project properly builds. There may be issues with the internal build, will deal with it there, or will revert these ignores in case it can't be dealt with